crates/rolldown/tests/esbuild/loader/loader_data_url_base64_invalid_utf8/artifacts.snap

```diff
@@ -8,7 +8,7 @@ source: crates/rolldown_testing/src/integration_test.rs
 ```js
 
 //#region binary.txt
-var binary_default = "data:text/plain;base64,/w==";
```

Comment:
This looks regressed,because before we use extension to determine `.txt`'s mime, this did not align with `esbuild`

Reply:
https://github.com/evanw/esbuild/blob/d34e79e2a998c21bb71d57b92b0017ca11756912/internal/bundler/bundler.go#L769-L777